frontend/nuxt.config.ts

```diff
@@ -33,7 +33,7 @@ export default defineNuxtConfig({
         }
     },
     devServer: {
-        port: 80,
-        host: "0.0.0.0",
```

Comment:
@andreasgangso @KillerX I need this change to be able to open it on Chrome locally. But does it break the docker container?

Reply:
this is devserver so it doesnt matter.
but im getting :3000 when i run pnpm dev 🤷

Reply:
maybe something else is already using 8001?